vision_datasets/data_tasks/image_text_matching/manifest.py

```diff
@@ -0,0 +1,16 @@
+import pathlib
+import typing
+
+from ...data_manifest import ImageLabelManifest
+
+
+class ImageTextMatchingLabelManifest(ImageLabelManifest):
+    """
+    (text, match): where text is str, and match in [0, 1]
+    """
+
+    def __init__(self, label: typing.Tuple[str, int], label_path: pathlib.Path = None, additional_info: typing.Dict = None):
+        assert label[0]
```

Comment:
probably better to make a method to validate the labels, instead of adding asserts in init.